src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -10,6 +10,22 @@
 namespace miopen {
 namespace conv {
 
+static inline uint32_t igemm_find_tile_size_with_upper_bound(
+    uint32_t out_size, size_t upper_bound, uint32_t stride, uint32_t dilation, uint32_t filter)
+{
+    // return tile size so that the required input tile(sec_in) is no larger than upper_bound
+    uint32_t n_tiles = 1;
+    for(; n_tiles <= out_size; n_tiles++)
```

Comment:
What if `Assuming 'n_tiles' is > 'out_size'`? i.e. out_size input is zero? I know that's the unlikely case but we need to guard it
ref: http://micimaster.amd.com/blue/rest/organizations/jenkins/pipelines/MLLibs/pipelines/MIOpen/branches/PR-1523/runs/1/nodes/14/log/?start=0

Reply:
Thanks for tracking CI, it will be updated in next commit.